### ISSUE TITLE
An error has occurred - After upgrade to 22.2.2

### ISSUE THREAD

**Issue by Jawn78** [SergeyMosin/Appointments#253]
**Nextcloud Version**: 22.2.2
**System:** Ubuntu
**DB:** MariaDB
**PHP:** PHP8.0-FPM PHP8.0

Go to set an appontment and met with 

```
An error has occurred
Please try again later
Thank you
```

**Comment by SergeyMosin** [comment]
I can not reproduce with:
```
Nextcloud Version: 22.2.3
Appointments Version: 1.11.2
```
Unfortunately I do not have NC22.2.2 to test on.

**Comment by Jawn78** [comment]
I will try to update to the newest patch 22.2.3 from 22.2.2 and circle back.

**Comment by Jawn78** [comment]
I was able to identify the issue. I am using a CalDav Synchronization tool that was using the globalappointmentID for UID attribute. This attribute was not able to write to the database length, but apparently would still work enough to create the calendar event which was still able to be seen by the appointment app. Then when someone would try to schedule an appointment, the appointment app would run into the issue again.

**Comment by Jawn78** [comment]
Fixed with 1.11.3  Thanks Sergey!
